lib/rails_admin_import/config.rb

```diff
@@ -1,52 +1,24 @@
-require 'rails_admin_import/config/model'
-# require 'active_support/core_ext/class/attribute_accessors'
+require 'rails_admin_import/config/legacy_model'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.